### PR TITLE
[react-native-material-ui] Correctly type style prop

### PR DESCRIPTION
**Pull request by sstur**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts#L1826
Also, notice that styles can be an array of styles according to docs [here](https://facebook.github.io/react-native/docs/style).
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by sstur** [comment]
Hey TypeScript folks. Maybe you can help me understand why this lint error gets thrown:
screen shot 2018-09-06 at 21 51 25

The relevant lines are:
screen shot 2018-09-06 at 21 52 29

Any reason why I shouldn't / can't make a local type alias? I definitely don't want to export those two aliases.

File: types/react-native-material-ui/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 // Definitions by: Kyle Roach <https://github.com/iRoachie>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
 import { Component } from 'react';
-import { ViewStyle, TextStyle, Image } from 'react-native';
+import { StyleProp, ViewStyle as ViewStyleRaw, TextStyle as TextStyleRaw, Image } from 'react-native';
+
+type ViewStyle = StyleProp<ViewStyleRaw>;
+type TextStyle = StyleProp<TextStyleRaw>;
```
Comment (iRoachie) [inline types/react-native-material-ui/index.d.ts]: All types/interfaces have to be exported so change to `export type`

**Comment by iRoachie** [comment]
@alloy Can you restart the ci?

**Pull request merged**